test/core/visualization/imageComparison/wavysurface.sat

```diff
@@ -0,0 +1,236 @@
+400 0 1 0        

+7 Unknown 13 ACIS 5.0.1 NT 24 Thu Jul 23 16:01:37 2015


```

Comment:
why is the sat file in here? Does the .dyn load this SAT or create the surface from scratch?

Reply:
the dyn load this SAT